src/types/observability.go

```diff
@@ -94,39 +94,45 @@ type NetworkSummary struct {
 	UpdatedTime          int64  `json:"UpdatedTime,omitempty"`
 	Count                int32  `json:"Count,omitempty"`
 }
+type PodOwner struct {
+	Ref       string `json:"Ref,omitempty"`
+	Name      string `json:"Name,omitempty"`
+	Namespace string `json:"Namespace,omitempty"`
+}
```

Comment:
```suggestion
type Workload struct {
	Type       string `json:"Type,omitempty"`
	Name      string `json:"Name,omitempty"`
}
```
- We don't need `Namespace` here since we already have `SystemSummary.NamespaceName`.